cloud: update region CIDR docs (#16132)
pingcap/docs: tidb-cloud/set-up-vpc-peering-connections.md

VPC peering connection is a networking connection between two VPCs that enables you to route traffic between them using private IP addresses. Instances in either VPC can communicate with each other as if they are within the same network.

Currently, TiDB clusters of the same project in the same region are created in the same VPC. Therefore, once VPC peering is set up in a region of a project, all the TiDB clusters created in the same region of this project can be connected in your VPC. VPC peering setup differs among cloud providers.

> **Tip:**
>
> To connect your application to TiDB Cloud, you can also set up [private endpoint connection](/tidb-cloud/set-up-private-endpoint-connections.md) with TiDB Cloud, which is secure and private, and does not expose your data to the public internet. It is recommended to use private endpoints over VPC peering connections.

> ## Prerequisite: Set a CIDR for this region

Reviewer: ```suggestion
## Prerequisite: Set a CIDR for a region
```

CIDR (Classless Inter-Domain Routing) is the CIDR block used for creating VPC for TiDB Dedicated clusters.

> Before adding VPC Peering requests to a region, you must set a CIDR for the region and create the initial TiDB Dedicated cluster in that region. Once the first Dedicated cluster is created, the VPC of the cluster will be created, allowing you to establish a peering link to your application's VPC.

Reviewer: ```suggestion
Before adding VPC Peering requests to a region, you must set a CIDR for that region and create an initial TiDB Dedicated cluster in that region. Once the first Dedicated cluster is created, TiDB Cloud will create the VPC of the cluster, allowing you to establish a peering link to your application's VPC.
```

> You can set the CIDR when creating the first Dedicated cluster. If you want to set the CIDR before creating the cluster, perform the following operations:

Reviewer: ```suggestion
You can set the CIDR when creating the first TiDB Dedicated cluster. If you want to set the CIDR before creating the cluster, perform the following operations:
```

1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
2. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Project Settings**.
3. On the **Project Settings** page of your project, click **Network Access** in the left navigation pane, and then click the **Project CIDR** tab.

> 4. Click **Create CIDR**, then click **AWS CIDR** or **Google Cloud CIDR** according to your cloud provider. Specify the Region and CIDR value in the **Create AWS CIDR** or **Create Google Cloud CIDR** window, and then click **Confirm**.

Reviewer: ```suggestion
4. Click **Create CIDR**, and then click **AWS CIDR** or **Google Cloud CIDR** according to your cloud provider. Specify the region and CIDR value in the **Create AWS CIDR** or **Create Google Cloud CIDR** window, and then click **Confirm**.
```

    > **Note:**
    >

>     > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. It is recommended to configure an IP range size between /16 and /23. The following network addresses are supported:

Reviewer: ```suggestion
    > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. It is recommended to configure an IP range size between `/16` and `/23`. Supported network addresses include:
```

>     > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. It is recommended to configure an IP range size between /16 and /23. The following network addresses are supported:

Reviewer 2: For GCP, the IP range size is between /16 and /21.

>     > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. It is recommended to configure an IP range size between `/16` and `/23`. Supported network addresses include:

Reviewer 3: ```suggestion
    > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. 
    > For AWS Region, it is recommended to configure an IP range size between `/16` and `/23`; for Google Cloud Region, it is recommended to configure an IP range size between `/16` and `/21`.Supported network addresses include:
```

Before adding VPC Peering requests to a region, you must set a CIDR for that region and create an initial TiDB Dedicated cluster in that region. Once the first Dedicated cluster is created, TiDB Cloud will create the VPC of the cluster, allowing you to establish a peering link to your application's VPC.

You can set the CIDR when creating the first TiDB Dedicated cluster. If you want to set the CIDR before creating the cluster, perform the following operations:

1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
2. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Project Settings**.
3. On the **Project Settings** page of your project, click **Network Access** in the left navigation pane, and then click the **Project CIDR** tab.
4. Click **Create CIDR**, and then click **AWS CIDR** or **Google Cloud CIDR** according to your cloud provider. Specify the region and CIDR value in the **Create AWS CIDR** or **Create Google Cloud CIDR** window, and then click **Confirm**.

    > **Note:**
    >

>     > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. 
>     > For AWS Region, it is recommended to configure an IP range size between `/16` and `/23`; for Google Cloud Region, it is recommended to configure an IP range size between `/16` and `/21`.Supported network addresses include:

Author: ```suggestion
    > - To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. 
    > - For AWS Region, it is recommended to configure an IP range size between `/16` and `/23`. For Google Cloud Region, it is recommended to configure an IP range size between `/16` and `/21`. Supported network addresses include:
```
